flask-hello-world-app/app.py

```diff
@@ -1,5 +1,9 @@
 import pandas as pd
 from flask import Flask
+import logging
+
+log = logging.getLogger('werkzeug')
+log.setLevel(logging.ERROR)
```

Comment:
What's the purpose of this? Does this make all logs at the ERROR level? Is this left over from debugging?

Reply:
This is a fix from Andre. Basically before it would give logging messages that looked like errors and customers were getting confused. This prevents the messages from going to logs unless they are errors.